tasks/scrapeData.js

```diff
@@ -133,9 +133,17 @@ function addData(cases, location, result) {
   Run the correct scraper for this location
 */
 function runScraper(location) {
+  const rejectUnauthorized = location.certValidation === false;
+  if (rejectUnauthorized) {
+    // Important: this prevents SSL from failing
+    process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+  }
   if (typeof location.scraper === 'function') {
     return location.scraper();
   }
+  if (rejectUnauthorized) {
+    delete process.env.NODE_TLS_REJECT_UNAUTHORIZED;
+  }
```

Comment:
@qgolsteyn I prefer this approach -- that way we have the information surfaced for ratings. Thoughts?

Reply:
That said, it's not as easily testable...

Reply:
I feel both can exist? Although I could also modify the test to set `process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0"` before it runs as well, so testing is a non-issue.

Reply:
Yeah, it's fine if both exist...